apps/meta-data-utils/package.json

```diff
@@ -21,7 +21,8 @@
     "test-ci": "vitest run",
     "build": "vite build",
     "preview": "vite preview",
-    "format": "prettier src test --write --config ../.prettierrc.js"
+    "format": "prettier src test --write --config ../.prettierrc.js",
```

Comment:
would be nice if we could have a single script with params to avoid these issue , but this seems to work